.pre-commit-config.yaml

```diff
@@ -32,11 +32,6 @@ repos:
     # run this hook on `labeler.yml`.
     exclude: .github/labeler.yml
 
-- repo: https://github.com/tox-dev/pyproject-fmt
-  rev: 0.9.2
-  hooks:
-  - id: pyproject-fmt
-
```

Comment:
I'm removing this pre-commit hook since it was doing things like changing `2.5.0` → `2.5`, where my goal is to have exact versions for the minimum requirements for when we automate it.  This hook doesn't have a way to turn it off though.

Reply:
:thumbsup: